[2021-12-01 16:46:42,160] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map3 manual__2021-12-01T15:00:12.054797+00:00 [queued]>
[2021-12-01 16:46:42,172] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: FrequencyCount.map3 manual__2021-12-01T15:00:12.054797+00:00 [queued]>
[2021-12-01 16:46:42,172] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:46:42,172] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-12-01 16:46:42,172] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-01 16:46:42,185] {taskinstance.py:1262} INFO - Executing <Task(MapOperator): map3> on 2021-12-01 15:00:12.054797+00:00
[2021-12-01 16:46:42,190] {standard_task_runner.py:52} INFO - Started process 148 to run task
[2021-12-01 16:46:42,194] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'FrequencyCount', 'map3', 'manual__2021-12-01T15:00:12.054797+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/mapred.py', '--cfg-path', '/tmp/tmpcamudpir', '--error-file', '/tmp/tmp8ielwfwo']
[2021-12-01 16:46:42,194] {standard_task_runner.py:77} INFO - Job 57: Subtask map3
[2021-12-01 16:46:42,226] {logging_mixin.py:109} INFO - Running <TaskInstance: FrequencyCount.map3 manual__2021-12-01T15:00:12.054797+00:00 [running]> on host 60ba21408f08
[2021-12-01 16:46:42,270] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=FrequencyCount
AIRFLOW_CTX_TASK_ID=map3
AIRFLOW_CTX_EXECUTION_DATE=2021-12-01T15:00:12.054797+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-01T15:00:12.054797+00:00
[2021-12-01 16:46:42,499] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=FrequencyCount, task_id=map3, execution_date=20211201T150012, start_date=20211201T164642, end_date=20211201T164642
[2021-12-01 16:46:42,526] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-01 16:46:42,576] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
